test/utils/isSameDay_spec.js

```diff
@@ -15,6 +15,15 @@ describe('isSameDay', () => {
     expect(isSameDay(today, tomorrow)).to.equal(false);
   });
 
+  it('returns false for same days of week', () => {
+    // Flags accidentally use of moment's day() function, which returns index
```

Comment:
The naming in JS Dates is "day" means "day of week" and "date" means "calendar date" - if moment or we deviate from that, it's a problem that needs fixing.